Cannot find module './src/error.implementation'

# Main description

Failed to run example implementation for Protocol interfaces.

# Error output

```
internal/modules/cjs/loader.js:589
    throw err;
    ^

Error: Cannot find module './src/error.implementation'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:587:15)
    at Function.Module._load (internal/modules/cjs/loader.js:513:25)
    at Module.require (internal/modules/cjs/loader.js:643:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/xromash/Documents/Programming/nodejs/Group3_IoT_Services/Utils/src/config.profile.js:1:99)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
```

--- a/Utils/src/config.profile.js
+++ b/Utils/src/config.profile.js
@@ -1,4 +1,4 @@
-const AbstractNotImplementedError = require("./src/error.implementation").AbstractNotImplementedError
+const AbstractNotImplementedError = require("./error.implementation").AbstractNotImplementedError
 
 class ConfigurationProfile {
     constructor(profileClass) {

